dataflow/src/main/java/org/checkerframework/dataflow/busyexpression/BusyExprStore.java

```diff
@@ -0,0 +1,187 @@
+package org.checkerframework.dataflow.busyexpression;
+
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.checkerframework.dataflow.analysis.Store;
+import org.checkerframework.dataflow.cfg.node.IntegerDivisionNode;
+import org.checkerframework.dataflow.cfg.node.Node;
+import org.checkerframework.dataflow.cfg.node.NumericalAdditionNode;
+import org.checkerframework.dataflow.cfg.node.NumericalMultiplicationNode;
+import org.checkerframework.dataflow.cfg.node.NumericalSubtractionNode;
+import org.checkerframework.dataflow.cfg.visualize.CFGVisualizer;
+import org.checkerframework.dataflow.expression.JavaExpression;
+import org.checkerframework.javacutil.BugInCF;
+
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.LinkedHashSet;
+import java.util.Set;
+import java.util.StringJoiner;
+
+/** A busy expression store contains a set of busy expressions represented by nodes. */
+public class BusyExprStore implements Store<BusyExprStore> {
+
+    /** A set of busy expression abstract values. */
+    private final Set<BusyExprValue> busyExprValueSet;
+
+    /**
+     * Create a new BusyExprStore.
+     *
+     * @param busyExprValueSet a set of busy expression abstract values
+     */
+    public BusyExprStore(Set<BusyExprValue> busyExprValueSet) {
+        this.busyExprValueSet = busyExprValueSet;
+    }
+
+    /** Create a new BusyExprStore. */
+    public BusyExprStore() {
+        busyExprValueSet = new LinkedHashSet<>();
+    }
+
+    /**
+     * Kill expressions if they contain variable v.
+     *
+     * @param v a variable
+     */
+    public void killBusyExpr(Node v) {
```

Comment:
Sure, I will change parameter name for consistency.
But I think `killBusyExpr` is ok, because `kill` represents the kill set in the transfer function. And `live-variable` example also uses `kill-xx`.